plugins/node/opentelemetry-instrumentation-cassandra/test/cassandra-driver.test.ts

```diff
@@ -228,6 +253,38 @@ describe('CassandraDriverInstrumentation', () => {
         assertSingleSpan('cassandra-driver.execute', query.substr(0, 25));
       });
     });
+
+    describe('responseHook', () => {
+      const customAttribute = {
+        attributeName: 'custom-attribute',
+        attributeValue: 'response-value',
+      };
+
+      before(() => {
+        const config: CassandraDriverInstrumentationConfig = {
+          responseHook: (span, response) => {
+            span.setAttribute(
```

Comment:
Could be nice to also add an attribute from `response`, to validate hook populate it as expected